Configuration/TCA/tx_telephonedirectory_domain_model_languageskill.php

```diff
@@ -1,4 +1,28 @@
 <?php
+
+$languageSkills = [
+    [
+        'LLL:EXT:telephonedirectory/Resources/Private/Language/locallang_db.xlf:tx_telephonedirectory_domain_model_languageskill.skill.0',
+        '0'
+    ],
+    [
+        'LLL:EXT:telephonedirectory/Resources/Private/Language/locallang_db.xlf:tx_telephonedirectory_domain_model_languageskill.skill.1',
+        '1'
+    ],
+    [
+        'LLL:EXT:telephonedirectory/Resources/Private/Language/locallang_db.xlf:tx_telephonedirectory_domain_model_languageskill.skill.2',
+        '2'
+    ],
+    [
+        'LLL:EXT:telephonedirectory/Resources/Private/Language/locallang_db.xlf:tx_telephonedirectory_domain_model_languageskill.skill.3',
```

Comment:
why do you have changed all language keys back to integers. Do you know in 5 year what they mean?